### PR TITLE
Update About screen with components from Ark-android

### PR DESCRIPTION
**Pull request by hieuwu**
Update About screen with components from Ark-android
Upgrade libraries
arkandroid

### CODE REVIEW

File: app/src/main/java/dev/arkbuilders/rate/presentation/settings/AboutScreen.kt [hunk 1]
```diff
@@ -50,275 +22,12 @@ fun AboutScreen(navigator: DestinationsNavigator) {
...
-            modifier = Modifier.padding(horizontal = 16.dp),
-            horizontalAlignment = Alignment.CenterHorizontally,
-        ) {
-            Icon(
-                modifier = Modifier.padding(top = 32.dp),
-                painter = painterResource(id = R.drawable.ic_about_logo),
-                contentDescription = "",
-                tint = Color.Unspecified,
+            ArkAbout(
+                appName = stringResource(id = R.string.app_name),
```
Comment (mdrlzy) [inline app/src/main/java/dev/arkbuilders/rate/presentation/settings/AboutScreen.kt:26]: Oh, I forgot to add modifier arg)

**Review by mdrlzy — CHANGES_REQUESTED** [review]
Please remove unused resources. Also check strings
They were added in this [commit](https://github.com/ARK-Builders/ARK-Rate/pull/96/commits/e2eddba5b0d42d4cad53a2072300c1c2f938c757#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103)

**Review by kirillt — APPROVED** [review]

File: app/src/main/java/dev/arkbuilders/rate/presentation/settings/AboutScreen.kt (hunk 1 above)
Comment (hieuwu) [reply app/src/main/java/dev/arkbuilders/rate/presentation/settings/AboutScreen.kt:26]: I remove unused resources in https://github.com/ARK-Builders/ARK-Rate/pull/108/commits/d7373dd2c3cecc6d3b259f0ace02874f54f6a316
I'll add modifier later

**Review by mdrlzy — APPROVED** [review]
Good job! 
Bump `ark-about` to `0.1.1`, use `modifier` inside `ArkAbout` instead of `Box` and feel free to merge.

**Pull request merged**